src/query/expression/Cargo.toml

```diff
@@ -26,6 +26,7 @@ common-jsonb = { path = "../../common/jsonb" }
 educe = "0.4"
 enum-as-inner = "0.5"
 enum_dispatch = "0.3.8"
+ethnum = { version = "1.3", features = ["serde"] }
```

Comment:
Seems this crate is better than `primitive_types`.

https://github.com/nlordell/ethnum-rs/blob/d8a2c42f7bd4c06d076a7bbcbaaaa948c67d2a22/bench/benches/num.rs